wire/internal/wire/wire.go

```diff
@@ -60,6 +61,16 @@ func (gen GenerateResult) Commit() error {
 	return ioutil.WriteFile(gen.OutputPath, gen.Content, 0666)
 }
 
+// Diff writes a diff of the current file on disk vs the generated file to out.
+func (gen GenerateResult) Diff() (string, error) {
```

Comment:
I would prefer this function and dependency be moved to the CLI binary rather than the library.

Reply:
Done.